tests/general/MementoBearTest.py

```diff
@@ -283,9 +284,6 @@ def test_settings_follow_redirects(self):
 
     def test_links_to_ignore(self):
         valid_file = """
-        # web.archive.org should be ignored
-        http://web.archive.org/web/*/spiegel.com
```

Comment:
Ok, when this is merged, create a new issue to create a new default ignore list which is only used by MementoBear